linux/Platform.c

```diff
@@ -66,6 +66,7 @@ in the source distribution for its full text.
 
 #ifdef HAVE_SENSORS_SENSORS_H
 #include "LibSensors.h"
+#include <errno.h>
```

Comment:
Order? Is it only used with `HAVE_SENSORS_SENSORS_H` active?

Reply:
After seeing [lgtm](https://lgtm.com/projects/g/htop-dev/htop/logs/rev/pr-14d11c36ce175c408cc1ff7a16a95e0c50a49c85/lang:cpp/stage:Build%20merge_628a5fa050b2d91cdd8989404e4aa1f57f23e852) log. I thought I was supposed to use ```errno.h``` when ```HAVE_SENSORS_SENSORS_H``` is active. On furthur inspection I found that it is supposed to be active when ```HAVE_LIBCAP``` is active.